cisagov/findcdn
# Code cleanup

# 

## 🗣 Description



Refactoring code base to better follow python PEP-8 convention to improve future maintainability.

## 💭 Motivation and Context



Following issue #14, the code base should more strictly follow Python convention to hopefully improve maintainability. 

## 🧪 Testing




Since no new functionality was added to the project, we used the same tests available in the repository to verify proper functionality. Changes to the tests are artifacts of changing how variables need to be passed to specific methods or classes in the project. 
### detectcdn
For this sub-module we tested all functions by running them on working and non-working domains.
* IP resolution function
* WHOIS function
*  CNAME function
* HTTPS function
* All checks function
### cdnengine
For this we test if the cdnengine can properly handle sets of domains:
* Test chef object initialization 
* Test grab_cdn with set of domains with definite CDNs
* Test has_cdn with domains that have CDNs and domains with no CDN
* Test that the basics of importing domains works for chef. 
### findcdn
We test in this module the ability for us to validate and pass information to cdnengine as well as exporting information:
* Test basic list works to pass to cdnengine
* Make sure our verbose setting is working
* Test if detection of fake or broken domains works
* Check if our input or outfiles exist or do not exist 

## 🚥 Types of Changes




- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist
